pkg/apiserver/webhooks/datavolume-mutate.go

```diff
@@ -101,17 +101,19 @@ func (wh *dataVolumeMutatingWebhook) Admit(ar admissionv1.AdmissionReview) *admi
 	modifiedDataVolume := dataVolume.DeepCopy()
 	modified := false
 
-	config, err := wh.cdiClient.CdiV1beta1().CDIConfigs().Get(context.TODO(), common.ConfigName, metav1.GetOptions{})
-	if err != nil {
-		return toAdmissionResponseError(err)
-	}
-	if config.Spec.DataVolumeTTLSeconds != nil &&
-		modifiedDataVolume.Annotations[controller.AnnDeleteAfterCompletion] != "false" {
-		if modifiedDataVolume.Annotations == nil {
-			modifiedDataVolume.Annotations = make(map[string]string)
+	if ar.Request.Operation == admissionv1.Create {
```

Comment:
it now only applies on create, does it mean previously it could apply to update or something like that?
Maybe it is worth to state it in commit message, instead of "minor fixes".

Reply:
it's not that dramatic, but I'll add it to the commit message